client/my-sites/plans/ecommerce-trial/entrepreneur-plan/style.scss

```diff
@@ -0,0 +1,99 @@
+@import "@automattic/onboarding/styles/mixins";
+
+.entrepreneur-trial-plan__included-wrapper {
+	display: flex;
+	flex-flow: row wrap;
+	margin-top: 30px;
+	margin-left: -12px;
+	margin-right: -12px;
+}
+.entrepreneur-trial-plan__section-title {
+	font-size: 1.5rem;
```

Comment:
I wonder if we don't have predefined variable for this size.

Reply:
Got it, it is called `font-title-medium`